vehicle/mkvi/software/bms/bms.c

```diff
@@ -225,6 +263,15 @@ int main(void) {
             //         loop_counter = 0;
             //     }
 
+            if (can_poll_receive_dashboard() == 0) {
+                can_receive_dashboard();
+
+                bool ready_to_drive = dashboard.ready_to_drive;
```

Comment:
why do you need to create a separate variable instead of directly using `dashboard.ready_to_drive`